app/page.tsx

```diff
@@ -25,7 +25,8 @@ import { LuFilter } from "react-icons/lu";
 import NoEvents from "./NoEvents";
 import FCM from "@/components/ui/fcm";
 import moment from "moment";
-import CountdownTimer from "@/components/ui/countdown";
+import CountDown from "@/components/ui/CountDown";
```

Comment:
Fix this import

File name's wrong.

Reply:
Actually, I had changed the file name but it's not showing in the commit